package.json

```diff
@@ -1,35 +1,28 @@
 {
-  "name": "excel-custom-functions",
-  "version": "1.0.0",
-  "description": "Create Excel functions using JavaScript.",
-  "main": "customfunctions.js",
+  "name": "excel-functions",
+  "version": "0.0.2",
+  "main": "./src/customfunctions.ts",
+  "typings": "lib/index.d.ts",
   "scripts": {
-    "start": "webpack-dev-server --mode development --https --key ./certs/server.key --cert ./certs/server.crt --cacert ./certs/ca.crt --port 3000 --hot --inline",
-    "sideload": "office-toolbox sideload -m manifest.xml -a Excel",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "webpack -p --mode production",
+    "sideload": "office-toolbox sideload -m manifest.xml -a excel",
+    "start": "start npm run webpack && start npm run dev-server && npm run sideload",
+    "dev-server": "webpack-dev-server --mode development --https --key ./certs/server.key --cert ./certs/server.crt --cacert ./certs/ca.crt --port 8081 --hotOnly",
+    "webpack": "webpack -p --mode development --watch"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/OfficeDev/Excel-Custom-Functions.git"
-  },
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/OfficeDev/Excel-Custom-Functions/issues"
-  },
-  "homepage": "https://github.com/OfficeDev/Excel-Custom-Functions#readme",
   "devDependencies": {
-    "typescript": "^2.8.1",
-    "office-toolbox": "^0.1.0",
-    "@types/office-js": "^0.0.37",
-    "@types/jquery": "^2.0.39",
     "file-loader": "^1.1.11",
+    "clean-webpack-plugin": "^0.1.19",
     "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^3.0.7",
+    "html-webpack-plugin": "^3.2.0",
+    "http-server": "^0.11.1",
     "office-addin-validator": "^1.0.1",
```

Comment:
I don't think we need office-addin-validator.

Reply:
Looks like we do need it for sideloading. When I remove it, sideloading fails: 
"Error: Cannot find module 'office-addin-validator'"